recipes-soletta/dev-app/soletta-dev-app_0.0.7.bb

```diff
@@ -2,25 +2,21 @@ DESCRIPTION = "Soletta Development Application"
 DEPENDS = "nodejs-native"
 RDEPENDS_${PN} = "soletta nodejs systemd graphviz libmicrohttpd avahi-daemon bash git"
 LICENSE = "Apache-2.0"
-PV = "1_beta8+git${SRCPV}"
 
 LIC_FILES_CHKSUM = "file://${S}/LICENSE;md5=93888867ace35ffec2c845ea90b2e16b"
 
-SRC_URI = "git://git@github.com/solettaproject/soletta-dev-app.git;protocol=https \
+SRC_URI = "npm://registry.npmjs.org;name=${PN};version=${PV} \
            file://soletta-dev-app.service \
            file://soletta-dev-app-mac.sh \
            file://soletta-dev-app-avahi-discover.service \
```

Comment:
You should consider using the shrinkwrap and lockdown features of npm-fetcher. These would lead to certain versions of the dependencies being fetched, and thus to reproducible builds.